az-core/src/main/java/azkaban/Constants.java

```diff
@@ -480,4 +481,15 @@
     // common private properties for multiple plugins
     public static final String COMMONSYSCONFFILE = "commonprivate.properties";
   }
+
+  public static class ContainerizationProperties {
+    public static final String CONTAINERIZATION_IMPL_TYPE = "azkaban.containerization.impl"
+        + ".type";
+    public static final String CONTAINERIZATION_EXECUTION_BATCH_ENABLED = "azkaban"
+        + ".containerization.execution.batch.enabled";
```

Comment:
Just a thought: "Containerization" is the project name but is not reflective of what you are doing here. Should something like: ContainerDispatchProperties or something else a better name? Also corresponding names in the Key-Values of various properties?

Reply:
Thank you for reviewing PR and suggesting this change. I have fixed it and updated the PR.